gateway-js/src/datasources/RemoteGraphQLDataSource.ts

```diff
@@ -17,18 +17,21 @@ import { isObject } from '../utilities/predicates';
 import { GraphQLDataSource, GraphQLDataSourceProcessOptions, GraphQLDataSourceRequestKind } from './types';
 import createSHA from 'apollo-server-core/dist/utils/createSHA';
 import { parseCacheControlHeader } from './parseCacheControlHeader';
-
+import fetcher from 'make-fetch-happen';
 export class RemoteGraphQLDataSource<
   TContext extends Record<string, any> = Record<string, any>,
 > implements GraphQLDataSource<TContext>
 {
-  fetcher: typeof fetch = fetch;
+  fetcher: typeof fetch;
 
   constructor(
     config?: Partial<RemoteGraphQLDataSource<TContext>> &
       object &
       ThisType<RemoteGraphQLDataSource<TContext>>,
   ) {
+    this.fetcher = fetcher.defaults({
+      retry: false,
+    });
```

Comment:
This is already the default, but fine if you prefer to specify: https://github.com/npm/make-fetch-happen/blob/main/lib/options.js#L15
```suggestion
    this.fetcher = fetcher;
```

Edit: I just read the comment from @abernix about being very deliberate about this. If that's the case, I'd add a comment here so a person like me doesn't "clean this up" by removing it as I've suggested above.

Reply:
I wonder if we should open an issue to force-attempt `POST` retries (typically, a no no) on `query`s.  Users have definitely asked for retries here but maybe worth creating a conversation about it.  It's possible this is covered by https://github.com/apollographql/federation/issues/844, but I'm not sure if that's referring to `RemoteGraphQLDataSource`s or some internal Apollo agent stuff.